packages/odyssey-storybook/src/components/odyssey-mui/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -42,9 +42,11 @@ const storybookMeta: ComponentMeta<typeof RadioGroup> = {
     },
     label: {
       control: "text",
+      defaultValue: "Speed",
     },
     name: {
       control: "text",
+      defaultValue: "storybook-radio",
```

Comment:
Instead of putting these here, can we do this instead?:
```ts
const defaultProps = {
  label: "Speed",
  name: "storybook-radio",
}

const Error: Story = Template.bind({})
Error.args = {
  ...defaultProps,
  errorMessage: "blah",
}
```